itou/job_applications/migrations/0035_link_diagnoses.py

```diff
@@ -38,77 +38,78 @@
 """
 from django.db import migrations
 
+
 # Use imports (application defined models) instead of `apps.get_model()`
 # (migration defined models). This can bite us later e.g. when running
 # tests in a fresh database that runs all migrations again and again.
 # However, we really need access to Managers, QuerySets and properties.
-from itou.eligibility.models import EligibilityDiagnosis
-from itou.job_applications.models import JobApplication
+# from itou.eligibility.models import EligibilityDiagnosis
+# from itou.job_applications.models import JobApplication
 
 
 def move_data_forward(apps, schema_editor):
     """
     Link eligibility diagnoses to job applications.
     """
```

Comment:
Je suggère de dire explicitement pourquoi on commente toute la migration et de mettre un `pass` explicite.

```suggestion
    Link eligibility diagnoses to job applications.

    In the end, it has bitten us: raw `imports` instead of `apps.get_model()`
    failed after new fields were added to the `User` model through new
    migrations (see comment above).

    Everything after the `pass` statement is now commented on purpose.
    """

    pass
```

Reply:
OK !